### PR TITLE
bugfix/1147 - Callsigns in tutorial should update when switching airports

### LINKED ISSUE #1147: Callsigns in tutorial should update when switching airports
#### Issue Description
The callsigns used in the tutorial dialogue are taken from the airport that's selected when the sim first loads. If the airport is switched, the tutorial keeps the old callsigns, making it harder to follow.

Reported by darren on Slack

#### Steps to reproduce
- Load sim and switch to another airport
- Open tutorial and navigate to third section (`Departing aircraft`)
- Example aircraft will not be present in the current airport

### PR DESCRIPTION
**Pull request by felixscheffer**
Resolves #1147

### CODE REVIEW

File: src/assets/scripts/client/tutorial/TutorialView.js [hunk 1]
```diff
@@ -105,6 +105,7 @@ export default class TutorialView {
         this.$tutorialToggle = $(SELECTORS.DOM_SELECTORS.TOGGLE_TUTORIAL);
         this.$tutorialPrevious = this.$tutorialView.find(SELECTORS.DOM_SELECTORS.PREV);
         this.$tutorialNext = this.$tutorialView.find(SELECTORS.DOM_SELECTORS.NEXT);
+        this._onAirportChangeHandler = this.onAirportChange.bind(this);
```
Comment (n8rzz) [inline src/assets/scripts/client/tutorial/TutorialView.js]: you'll want to add a `._setupHandlers()` method that is called after `._init()` in the constructor.  Then move this assignment there.

File: src/assets/scripts/client/tutorial/TutorialView.js [hunk 2]
```diff
@@ -182,6 +185,22 @@ export default class TutorialView {
...
     }
 
+    /**
+     * Reloads the tutorial when the airport is changed.
+     *
+     * @for tutorialView
+     * @method onAirportChange
+     */
+    onAirportChange() {
+        const step = prop.tutorial.step;
```
Comment (n8rzz) [inline src/assets/scripts/client/tutorial/TutorialView.js]: what is `step` here?  is this the `currentStep`?  meaning the current position in the tutorial?

File: src/assets/scripts/client/tutorial/TutorialView.js (hunk 1 above)
Comment (felixscheffer) [reply src/assets/scripts/client/tutorial/TutorialView.js]: done

File: src/assets/scripts/client/tutorial/TutorialView.js (hunk 2 above)
Comment (felixscheffer) [reply src/assets/scripts/client/tutorial/TutorialView.js]: I decided to remove that code as it causes new issues. If you are switching to another airport, the tutorial should now reset to the first page.

**Pull request merged**